pkgs/applications/graphics/comical/default.nix

```diff
@@ -9,18 +9,31 @@ stdenv.mkDerivation rec {
     sha256 = "0b6527cc06b25a937041f1eb248d0fd881cf055362097036b939817f785ab85e";
   };
 
-  buildInputs = [ wxGTK util-linux zlib ];
-  makeFlags = [ "prefix=${placeholder "out"}" ];
+  patches = [ ./wxgtk-3.2.patch ];
 
-  patches = [ ./wxgtk-2.8.patch ];
+  buildInputs = [
+    wxGTK32
+    util-linux
+    zlib
+  ] ++ lib.optionals stdenv.isDarwin [
+    Cocoa
+  ];
+
+  makeFlags = [
+    "prefix=${placeholder "out"}"
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
Is this Darwin specific or just non-GCC? Does `"CC=$CC"` work?

Reply:
`CC=cc` does work. Updated the commit.